src/index.ts

```diff
@@ -392,23 +392,12 @@ export class EsbuildServerlessPlugin implements ServerlessPlugin {
       path.join(this.serviceDirPath, SERVERLESS_FOLDER)
     );
 
-    if (this.options.function) {
```

Comment:
We set `this.functions` to only include this function and filter it out if it is not a node function.

We can reuse the same code block with package individually to ignore it if it is not a node function.

https://github.com/floydspace/serverless-esbuild/blob/573385c5642a31d243b55004997e7c513a67104d/src/index.ts#L170-L175